tests/integration/ceph_mgr_test.go

```diff
@@ -122,19 +115,68 @@ func (s *CephMgrSuite) execute(command []string) (error, string) {
 	return s.installer.Execute("ceph", orchCommand, s.namespace)
 }
 
+func (s *CephMgrSuite) enableOrchestratorModule() {
+	logger.Info("Enabling Rook orchestrator module: <ceph mgr module enable rook --force>")
+	err, output := s.installer.Execute("ceph", []string{"mgr", "module", "enable", "rook", "--force"}, s.namespace)
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to enable rook orchestrator module: %q", err)
+	}
+
+	logger.Info("Setting orchestrator backend to Rook .... <ceph orch set backend rook>")
+	err, output = s.execute([]string{"set", "backend", "rook"})
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to set rook as backend orchestrator module: %q", err)
+	}
+}
+
 func (s *CephMgrSuite) waitForOrchestrationModule() {
 	var err error
+
+	// Status struct
+	type orchStatus struct {
+		Available bool   `json:"available"`
+		Backend   string `json:"backend"`
+	}
+
 	for timeout := 0; timeout < 30; timeout++ {
-		err, output := s.execute([]string{"status"})
+		logger.Info("Waiting for rook orchestrator module enabled and ready ...")
+		err, output := s.execute([]string{"status", "--format", "json"})
 		logger.Infof("%s", output)
 		if err == nil {
-			logger.Info("Rook Toolbox ready to execute commands")
-			return
+			logger.Info("Ceph orchestrator ready to execute commands")
+
+			// Get status information
+			bytes := []byte(output)
+			var status orchStatus
+			err := json.Unmarshal(bytes, &status)
+			if err != nil {
+				logger.Error("Error getting ceph orch status")
+				continue
+			}
+
+			if status.Backend != "rook" {
+				assert.Fail(s.T(), fmt.Sprintf("Orchestrator backend is <%q>. Setting it to <Rook>", status.Backend))
+				s.enableOrchestratorModule()
+			} else {
+				logger.Info("Orchestrator backend is <Rook>")
+				return
+			}
+		} else {
+			exitError, _ := err.(*exec.ExitError)
+			if exitError.ExitCode() == 22 { // The <ceph orch> commands are still not recognized
+				logger.Info("Ceph manager modules still not ready ... ")
+			} else if exitError.ExitCode() == 2 { // The rook orchestrator is not the orchestrator backend
+				s.enableOrchestratorModule()
+			}
 		}
 		time.Sleep(2 * time.Second)
```

Comment:
In the test log I see a bunch of retries, so a longer sleep between each try should be fine
```suggestion
		time.Sleep(5 * time.Second)
```